src/include/extension/catalog_extension.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include "catalog/catalog_content.h"
+
+namespace kuzu {
+namespace extension {
+
+class CatalogExtension : public catalog::CatalogContent {
```

Comment:
l assume you know what you are doing. 

But also keep in mind that we have a plan to remove CatalogContent. So depending on it at this time point is not ideal.

Reply:
Yes, for now we only have to extend the catalogcontent since we haven't implemented transaction for extension